src/rules/no-focused-tests.ts

```diff
@@ -71,7 +72,23 @@ export default createRule({
           callee.object.type === AST_NODE_TYPES.Identifier &&
           isCallToFocusedTestFunction(callee.object)
         ) {
-          context.report({ messageId: 'focusedTest', node: callee.object });
+          context.report({
+            messageId: 'focusedTest',
+            node: callee.object,
+            suggest: [
+              {
+                messageId: 'suggestRemoveFocus',
+                fix(fixer) {
+                  return [
+                    fixer.removeRange([
+                      callee.object.range[0],
+                      callee.object.range[0] + 1,
+                    ]),
+                  ];
```

Comment:
```suggestion
                  return fixer.removeRange([
                      callee.object.range[0],
                      callee.object.range[0] + 1,
                    ])
```

(prettier won't be happy, so run `yarn lint --fix` after applying all of these)